lib/build.gradle

```diff
@@ -0,0 +1,56 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
```

Comment:
why do you need maven-publish?

Reply:
Because I use this library in the second part of the task and connect it when implementing the method in the second part of the task.